src/types.db

```diff
@@ -37,6 +37,7 @@ count                   value:GAUGE:0:U
 counter                 value:COUNTER:U:U
 cpu                     value:DERIVE:0:U
 cpufreq                 value:GAUGE:0:U
+cpusleep                value:DERIVE:0:U
```

Comment:
I'd prefer to use the existing type `total_time_in_ms` which reports metrics in "milliseconds per second". We would need to divide the metric by 10^6 which is probably good enough.